_posts/2025-09-25-ethics-what-is-our-approach.md

```diff
@@ -0,0 +1,68 @@
+
+## Where we stand
+I'm sure it won't surprise anyone when we say that research ethics is incredibly important to us as a team and as a community! In some research settings, the term "ethics" doubles as "ensuring legal liabilities are met". In others, it might mean the difference between publishing in a peer-reviewed journal (with proof of an ethics process) or not being allowed to publish (without ethics). To OLS, it's probably both of those, as well as being trustworthy, thoughtful researchers. That means we _want_ to be responsible for our own work and behaviour, regardless of any other review processes we might take part in. Which leads to the next question! 
+
+## Looking for community input and opinions: What should we think about for OLS's formal research ethics process? 
+
+As a researcher, what would you like to see in a compassionate, carefully thought-out research ethics policy? What would you like us to avoid? Some examples based on discussion at our recent Seeds to Systems meeting: 
+
+### Avoid:
+- "Your form doesn't have a phone number on it. Please revise and resubmit with a phone number." (8 months later, the ethics review passes, once they've had a phone installed in the lab, even though they never intend to use it).
+- "One of your forms says last updated on yesterday's date and not today's date. Please revise and resubmit. "
+- "This is not human research, because it only involves talking to people, and doesn't involve any clinical interventions". (You can [read about the University of Minnesota getting banned from contributing to the linux kernel](https://www.theverge.com/2021/4/30/22410164/linux-kernel-university-of-minnesota-banned-open-source) if you'd like to understand the motivation for this one...)
+- Assuming that just because something is legal or compliant with terms of service, that it is therefore okay. 
+
+### Do: 
+- Think carefully about how you would feel if you were a research participant in a given situation. 
+- Review research protocols and data management plans (DMPs) as a team to make them as robust as possible, before submitting to the independent review board (IRB). 
+- Share experiences from different scientific domains and legal systems. 
+- Check that the research is as open as is possible whilst respecting other tensions such as privacy, conservation, and indigenous sovereignty.
+- Use open data formats wherever possible. 
+- Prepare consent forms in as plain-english (or other language) as possible.   
+
+## Where is the line - what requires ethics review and what doesn't? 
+Given our varying experiences with research ethics, and the way we enjoy working with community, we want to formalise exactly _which_ types of interaction and knowledge gathering should go through the ethics process and how much. Here's a hypothetical scale, from least strict to most strict: 
```

Comment:
>Here's a hypothetical scale, from least strict to most strict: 

Related to the previous comment around clarity on feedback request: Is this just hypothetical or an actual proposal we're asking feedback on?